projects/plugins/boost/app/modules/cache/data-sync/Page_Cache_Entry.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Automattic\Jetpack_Boost\Modules\Page_Cache\Data_Sync;
+
+use Automattic\Jetpack\WP_JS_Data_Sync\Contracts\Entry_Can_Get;
+use Automattic\Jetpack\WP_JS_Data_Sync\Contracts\Entry_Can_Set;
+use Automattic\Jetpack_Boost\Modules\Page_Cache\Pre_WordPress\Boost_Cache_Settings;
+
+class Page_Cache_Entry implements Entry_Can_Get, Entry_Can_Set {
+	public function get( $_fallback = false ) {
+		$cache_settings = Boost_Cache_Settings::get_instance();
+
+		$settings = array(
+			'bypassPatterns' => $cache_settings->get_bypass_patterns(),
```

Comment:
DataSync keys (any JSON keys) should be `snake_case` for consistency. Otherwise we have `window.jetpack_boost_ds.page_cache.bypassPatterns`

Reply:
Thanks!

Fixed in https://github.com/Automattic/jetpack/pull/35729/commits/073486060406b0b59dfe8c557ef2198b423f1141.